tool/src/org/antlr/v4/codegen/Target.java

```diff
@@ -689,6 +689,10 @@ public boolean supportsOverloadedMethods() {
 		return true;
 	}
 
+	public boolean ATNSerializedAsInts() {
+		return false;
```

Comment:
Probably it makes sense to return `true` by default because early you said `Ints` are more preferable than `Strings`.

Reply:
sounds good; I will reverse it